server/routes/users.js

```diff
@@ -116,10 +116,41 @@ export default (storage, scriptManager) => {
 
             return {
               user,
-              memberships: [ ]
+              memberships: []
             };
           });
       })
+      .then(data => {
+        if (!data.user.identities) {
+          data.connection = {};
+          return data;
+        }
+
+        const identities = data.user.identities.filter(identity => identity.provider === 'auth0');
+        const name = identities[0] && identities[0].connection;
+
+        if (!name) {
+          data.connection = {};
+          return data;
+        }
+
+        return req.auth0.connections.getAll({ name, fields: 'id' })
+          .then((connection) => {
+            if (connection && connection[0] && connection[0].id) {
+              return req.auth0.connections.get({ id: connection[0].id, fields: 'enabled_clients' });
+            }
+
+            return {};
+          })
+          .then((connection) => {
+            data.connection = {
+              ...connection,
```

Comment:
do we get, or need the ID here?

Reply:
There we have
```
id: "..."
is_domain_connection: false,
enabled_clients: [List],
name: "..."
```
we don't actually need anything but `enabled_clients`, so i'll remove unnecessary props.